python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -933,6 +972,10 @@ def run_transformer_autocast_bf16( repo_id,
     
     import pandas as pd
     for api in conf.test_api:
+        global arc_perf_test_counter
+        arc_perf_test_counter = 0
```

Comment:
I don't think it needs be a global variable?

Reply:
Will try to make it a local variable.